### ISSUE TITLE
Direct local .aar file dependencies are not supported when building an AAR

### ISSUE THREAD

**Issue by archcorsair** [adobe/react-native-aepassurance#13]
### Preface

The project built successfully previously. The only significant change was upgrading RN to version 0.64

### Expected Behavior

Project should compile

### Actual Behavior

Project does not compile

#### Steps to Reproduce

Build application in release mode

#### Platform and Version

macOS Big Sur: 11.1
React Native: 0.64
Android Studio: 4.1.3
`distributionUrl=https\://services.gradle.org/distributions/gradle-6.7-all.zip`

```
buildscript {
    ext {
        buildToolsVersion = "29.0.3"
        minSdkVersion = 21
        compileSdkVersion = 29
        targetSdkVersion = 29
        ndkVersion = "20.1.5948944"
    }
    repositories {
        google()
        jcenter()
    }
    dependencies {
        classpath('com.android.tools.build:gradle:4.0.1')

        // NOTE: Do not place your application dependencies here; they belong
        // in the individual module build.gradle files
    }
}
```

#### Sample Code that illustrates the problem

n/a

#### Logs taken while reproducing problem

```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':adobe_react-native-aepassurance:bundleReleaseAar'.
> Direct local .aar file dependencies are not supported when building an AAR. The resulting AAR would be broken because the classes and Android resources from any local .aar file dependencies would not be packaged in the resulting AAR. Previous versions of the Android Gradle Plugin produce broken AARs in this case too (despite not throwing this error). The following direct local .aar file dependencies of the :adobe_react-native-aepassurance project caused this error: /Users/<app path>/node_modules/@adobe/react-native-aepassurance/android/libs/aepcorebridge.aar
```

**Comment by shtomar-adb** [comment]
Hi @archcorsair ,
We have published a new npm package (v1.2.0) with this issue fixed. Please let us know if you still face the issue.

**Comment by archcorsair** [comment]
@shtomar-adb 
> We have published a new npm package (v1.2.0) with this issue fixed. Please let us know if you still face the issue.

Yes, it appears that v1.2.0 resolves this issue. Thank you for the fast turnaround on this issue.
